spec/models/work_package/work_package_custom_fields_spec.rb

```diff
@@ -256,7 +260,7 @@ def self.change_custom_field_value(work_package, value)
           change_custom_field_value(work_package, value)
         end
 
-        subject { work_package.journals.first.customizable_journals.first.value }
+        subject { work_package.journals.last.customizable_journals.first.value }
```

Comment:
There is no default order defined for journals. So the order in which the journals are returned is determined in the DB. You might be lucky of course but then again, you might not be.